runhouse/resources/hardware/sagemaker_cluster.py

```diff
@@ -1221,7 +1222,7 @@ def _stop_instance(self, delete_configs=True):
             rns_client.delete_configs(resource=self)
             logger.info(f"Deleted {self.name} from configs")
 
-    def _sync_runhouse_to_cluster(self, _install_url=None, env=None):
+    def _sync_runhouse_to_cluster(self, node: str, _install_url=None, env=None):
```

Comment:
Do we need to update the code below to use `node` ?

Reply:
yep good catch, was missing from the case where we install from site-packages